how-to/workspace-platform-starter/client/src/framework/shapes/browser-shapes.ts

```diff
@@ -92,6 +92,14 @@ export type BrowserProviderOptions = Pick<
 	 * The strategy for window positioning.
 	 */
 	windowPositioningStrategy?: CascadingWindowOffsetStrategy;
+
+	/**
+	 * By default we implement a Window Positioning Strategy that will try and position launched windows with an
+	 * offset. The way the windows are offset can be configured by specifying windowPositioningStrategy.
+	 * If you want to turn this off (e.g. automation tests that do not care about the layout of windows) then you
+	 * can set this value to true.
+	 */
+	disableWindowPositioningStrategy?: boolean;
```

Comment:
No need for this flag, if you have no `windowPositioningStrategy` it doesn't do any positioning

Reply:
Same reason as above.